app/javascript/packs/tabs.js

```diff
@@ -0,0 +1,51 @@
+import 'semantic-ui-sass';
+// Semantic
+$(() => {
```

Comment:
we are trying to get rid of `jQuery` nowadays, since vanilla js has all the needed functiionality already, so this approach is not the best idea
Also JS-classes are highly recommended